cost-analyzer/templates/_helpers.tpl

```diff
@@ -62,10 +62,6 @@ Create the fully qualified name for Prometheus alertmanager service.
 {{- end -}}
 {{- end -}}
 
-{{- define "cost-analyzer.serviceName" -}}
```

Comment:
We'd want to make the change in the cronjob template, to use cost-analyzer.serviceName, not in the helper/by changing all the other files-- so this should be left as-is and these should be changed: https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/templates/cost-analyzer-checks-template.yaml#L38